src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  line-height: 1;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
```

Comment:
it would be great to follow these suggestions: https://9elements.com/css-rule-order but it's not a huge issue